packages/babel-generator/src/buffer.js

```diff
@@ -99,6 +98,19 @@ export default class Buffer {
   }
 
   endsWith(str: string): boolean {
+    // Fast path to avoid iterating over this._queue.
+    if (str.length === 1) {
+      let last;
+      if (this._queue.length > 0) {
+        const str = this._queue[0][0];
```

Comment:
Ok first we need to remove kcheck and use eslint by itself with the preset I made https://github.com/babel/eslint-config-babel and we can just test it out to see if it's too many changes